.gcloudignore

```diff
@@ -0,0 +1 @@
+!.git
```

Comment:
Why is this necessary now? Could you add a comment?

Reply:
Done